### DISCUSSION TITLE
ANGULAR WHIT PHALCON

### DISCUSSION THREAD

**Discussion by GianJv** [phalcon/cphalcon#15909] [General]
I AM PLANNING TO MAKE AN IMPLEMENTATION OF ANGULAR FRONT AND BACKEND PHALCON, SOMEONE ALREADY HAD EXPERIENCE ON THE SUBJECT OR ANY RECOMMENDATION THAT THEY COULD MAKE, I AM ATTENTION TO THE COMMENTS

**Comment by niden** [comment]
Have a look at the `rest-api` repository. It will allow you to create an application that can "send" data back to Angular which will process them according to your application needs.
